v2/tweet_counts_options.go

```diff
@@ -0,0 +1,37 @@
+package twitter
+
+import (
+	"net/http"
+	"time"
+)
+
+// TweetRecentCountsOpts are the optional paramters that can be passed to the tweet recent counts callout
+type TweetRecentCountsOpts struct {
+	StartTime   time.Time
+	EndTime     time.Time
+	SinceID     string
+	UntilID     string
+	Granularity string
```

Comment:
It appears that the granularity is an enum.  Maybe instead of a string, there can be a type for this.
```
type Granularity string
const (
Hour Granularity = "hour"
// others
)

type TweetRecentCountsOpts struct {
    // others
    Granularity Granularity
}
```
This might help make the value implicit